content/200-concepts/100-components/02-prisma-client/index.mdx

```diff
@@ -67,7 +67,7 @@ You can install Prisma Client in your project with this command:
 npm install @prisma/client
 ```
 
-Note that this command also run the `prisma generate` command which generates the Prisma Client into the `node_modules/@prisma/client` directory.
+Note that this command also run the `prisma generate` command which generates the Prisma Client into the `node_modules/.prisma/client` directory.
```

Comment:
"into the `node_modules/.prisma/client` directory." should link to an in depth explanation here, or it is just confusing here (And should be left off).